pandas/tseries/offsets.py

```diff
@@ -579,9 +580,46 @@ class BusinessHourMixin(BusinessMixin):
 
     def __init__(self, start='09:00', end='17:00', offset=timedelta(0)):
         # must be validated here to equality check
-        start = liboffsets._validate_business_time(start)
+        if _iterable_not_string(start):
+            start = np.asarray(start)
```

Comment:
I would do this like

```
if not is_like_like(start):
    start = [start]
if not len(start):
   raise....
```

the np.asarray can be done when you validate

Reply:
Updated